src/combinatorics/simple_graph/coloring.lean

```diff
@@ -184,7 +184,7 @@ G.recolor_of_embedding $ (function.embedding.nonempty_of_card_le hn).some
 
 variables {G}
 
-lemma colorable.of_le {n m : ℕ} (hc : G.colorable n) (h : n ≤ m) : G.colorable m :=
+lemma colorable.of_le {n m : ℕ} (h : n ≤ m) (hc : G.colorable n) : G.colorable m :=
```

Comment:
```suggestion
lemma colorable.mono {n m : ℕ} (h : n ≤ m) (hc : G.colorable n) : G.colorable m :=
```
or
```suggestion
lemma colorable.mono_right {n m : ℕ} (h : n ≤ m) (hc : G.colorable n) : G.colorable m :=
```

Reply:
In what sense are you seeing these as being monotonic? I followed all the others, but these last two comments I'm finding to be non-obvious.

Reply:
`G.colorable` is monotone as a function `ℕ → Prop`. This is similar to eg `set.bounded.mono`.

Reply:
Ok, sounds good.